go.mod

```diff
@@ -34,6 +34,8 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.31.7
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.31.7
 	k8s.io/sample-controller => k8s.io/sample-controller v0.31.7
+	// Moved to the github.com/coder namespace: https://coder.com/blog/websocket
+	nhooyr.io/websocket => github.com/coder/websocket v1.8.7
```

Comment:
Why add a replace directive, compared to changing the import path in `pkg/agent/dispatcher.go`? 

That way `nhooyr.io/websocket`would not need to be mentioned anywhere in the codebase.

Reply:
I can also change the import there, but we need the replace directive to fix various other dependencies.